l5kit/setup.py

```diff
@@ -20,8 +20,8 @@
         "pymap3d",
         "scipy",
         "setuptools",
-        "torch==1.5.1",
-        "torchvision==0.6.1",
+        "torch>=1.5.0",
```

Comment:
this will install torch 1.6.0 which we haven't tested yet. Can we set it to >=1.5.0,<1.6.0?

Reply:
thanks! done